### PR TITLE
adding logs api example for elasticsearch

### PR DESCRIPTION
**Pull request by heeki**
*Description of changes:*
Adding Logs API example for sending logs to an Elasticsearch cluster

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python-example-logs-api-elasticsearch/extensions/logs_api_elasticsearch_extension/elasticsearch_producer.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import json
```
Comment (santiagocardenas) [inline python-example-logs-api-elasticsearch/extensions/logs_api_elasticsearch_extension/elasticsearch_producer.py]: Please add copyright notice.

**Comment by santiagocardenas** [comment]
I'm thinking it might be best to just use this one as a replacement of the original Elasticsearch example :)

File: python-example-logs-api-elasticsearch/extensions/logs_api_elasticsearch_extension/elasticsearch_producer.py (hunk 1 above)
Comment (heeki) [reply python-example-logs-api-elasticsearch/extensions/logs_api_elasticsearch_extension/elasticsearch_producer.py]: Added

**Comment by heeki** [comment]
> I'm thinking it might be best to just use this one as a replacement of the original Elasticsearch example :)

Done. Instead of creating a separate example, I've replaced the existing python-example-elasticsearch-extension with this new Logs API model.

File: python-example-elasticsearch-extension/README.md [hunk 2]
```diff
@@ -1,70 +1,41 @@
...
+After invoking the function and receiving the shutdown event, you should now see log messages from the example extension written back to the extension logs. For example:
 ```
-
-
-
-
-
+...
+BATCH RECEIVED: [{'time': '2020-11-12T00:30:48.883Z', 'type': 'platform.start', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c', 'version': '$LATEST'}}, {'time': '2020-11-12T00:30:48.993Z', 'type': 'platform.logsSubscription', 'record': {'name': 'logs_api_http_extension.py', 'state': 'Subscribed', 'types': ['platform', 'function']}}, {'time': '2020-11-12T00:30:48.993Z', 'type': 'platform.extension', 'record': {'name': 'logs_api_http_extension.py', 'state': 'Ready', 'events': ['INVOKE', 'SHUTDOWN']}}, {'time': '2020-11-12T00:30:49.017Z', 'type': 'platform.end', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c'}}, {'time': '2020-11-12T00:30:49.017Z', 'type': 'platform.report', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c', 'metrics': {'durationMs': 15.74, 'billedDurationMs': 100, 'memorySizeMB': 128, 'maxMemoryUsedMB': 62, 'initDurationMs': 226.3}}}]
```
Comment (santiagocardenas) [inline python-example-elasticsearch-extension/README.md]: You removed this output, right?

File: python-example-elasticsearch-extension/README.md [hunk 3]
```diff
@@ -1,70 +1,41 @@
...
-* `python-example-elasticsearch-extension/`: This sub-directory should be extracted to /opt/python-example-extension which is referenced by the `extensions/python-example-elasticsearch-extension` executable and includes a Python executable along with all of its necessary dependencies.
+In this example, we start by developing a simple extension and then add the ability to read logs from the Logs API. For more details on building an extension, please read the Extension API Developer Guide.
 
-## Prep Python Dependencies
-Install the extension dependencies locally, which will be mounted along with the extension code.
-
-```bash
-$ cd python-example-elasticsearch-extension
-$ chmod +x extension.py
-$ pip3 install -r requirements.txt -t .
```
Comment (santiagocardenas) [inline python-example-elasticsearch-extension/README.md:16]: There will be a need to pip install requirements, right?

Comment (heeki) [reply python-example-elasticsearch-extension/README.md:16]: Yes it requires the requests module.

File: python-example-elasticsearch-extension/README.md (hunk 2 above)
Comment (heeki) [reply python-example-elasticsearch-extension/README.md]: I have updated this to be an output from this extension.

File: python-example-elasticsearch-extension/README.md (hunk 3 above)
Comment (heeki) [reply python-example-elasticsearch-extension/README.md:16]: Removed the dependency on the requests module.

**Pull request merged**